src/client/components/app-core/links.css

```diff
@@ -0,0 +1,23 @@
+a {
+  color: var(--action-color);
+  text-decoration: none;
+}
+
+a:focus,
+a:hover {
+  border-bottom: 1px solid var(--action-color);
+}
+
+a[data-outbound]::after {
+  content: '';
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='512' height='512' viewBox='0 0 16 16'%3E%3Cpath fill='%236a9ce4' d='M14 16V5l-1 1v9H1V3h9l1-1H0v14z'/%3E%3Cpath fill='%236a9ce4' d='M16 0h-5l1.8 1.8L6 8.6 7.4 10l6.8-6.8L16 5z'/%3E%3C/svg%3E");
```

Comment:
Move to `/assets` and let Nuxt inline it? That way, its easier to preview and understand the icon file.